protocol/x/rewards/keeper/keeper.go

```diff
@@ -152,7 +153,7 @@ func (k Keeper) AddRewardSharesForFill(
 
 	// taker revshare is not returned if taker volume is greater than Max30dTakerVolumeQuantums
 	if value, ok := revSharesForFill.FeeSourceToRevSharePpm[revsharetypes.REV_SHARE_FEE_SOURCE_TAKER_FEE]; ok &&
```

Comment:
Shouldn't we remove `FeeSourceToRevSharePpm[revsharetypes.REV_SHARE_FEE_SOURCE_TAKER_FEE` from the if condition and only rely on ` fill.MonthlyRollingTakerVolumeQuantums < revsharetypes.MaxReferee30dVolumeForAffiliateShareQuantums ` only?

Reply:
my b  - didnt realize this when i was working through it fast